streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -313,16 +318,16 @@ public boolean conditionMet() {
                         final ReadOnlyWindowStore<String, Long> store = streamsWithKey.store(storeName, QueryableStoreTypes.<String, Long>windowStore());
                         return store != null && store.fetch(key, from, to) != null;
                     } catch (final IllegalStateException e) {
-                        // Kafka Streams instance may have closed but rebalance hasn't happened
+                        log.debug("Kafka Streams instance may have closed but re-balance hasn't happened", e);
                         return false;
                     } catch (InvalidStateStoreException e) {
-                        // there must have been at least one rebalance state
+                        log.debug("There must have been at least one rebalance state", e);
```

Comment:
This message is OK as a comment, but may be misleading as an debug log entry. If you do want to log it, maybe `The state store may has been closed already due to consecutive rebalances` is better. Ditto below.